src/popup.ts

```diff
@@ -147,6 +147,41 @@ async function init() {
     false
   );
 
+  document.body.addEventListener("keydown", (e) => {
+    if (
+      document.activeElement?.nodeName === "INPUT" ||
+      document.activeElement?.nodeName === "TEXTAREA"
+    ) {
+      return;
+    }
+
+    const index = Number(e.key);
+    if (Number.isNaN(index)) {
+      return;
+    }
+
+    const numEntries = instance.$store.state.accounts.entries.length;
+
+    if (index > numEntries) {
+      return;
+    }
+
+    const codeClipboard = document.querySelector<HTMLInputElement>(
+      "#codeClipboard"
+    );
+
+    if (!codeClipboard) {
+      return;
+    }
+
+    codeClipboard.value =
+      instance.$store.state.accounts.entries[index - 1].code;
```

Comment:
index here could be zero, and `index - 1` could be `-1`

Reply:
`instance.$store.state.accounts.entries` could not be shown by sort of the index in UI. Some accounts may be pinned, some accounts may be filtered.
Here's how are entries rendered: https://github.com/Authenticator-Extension/Authenticator/blob/dev/src/components/Popup/MainBody.vue#L26-L40